contracts/NFTs/Base/MultiPrivilege/MultiPrivilegeTransferableBurnable.sol

```diff
@@ -0,0 +1,229 @@
+//SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.13;
+
+import "./IMultiPrivilege.sol";
+import "../NftBaseUpgradeableTransferableBurnable.sol";
+
+/// @title MultiPrivilege
+/// @dev Based on the EIP-5496 https://eips.ethereum.org/EIPS/eip-5496
+abstract contract MultiPrivilege is
```

Comment:
@LorranSutter Is it common for these contract variants in `MultiPrivilegeTransferable.sol` and `MultiPrivilegeTransferableBurnable.sol` to have the same name as the "base"?